backend/src/Notifo.Domain.Integrations.Abstractions/MessagingMessage.cs

```diff
@@ -9,7 +9,17 @@ namespace Notifo.Domain.Integrations;
 
 public sealed class MessagingMessage : BaseMessage
 {
+    public IReadOnlyDictionary<string, string> Targets { get; set; }
+
     public string Text { get; set; }
+    public string? DetailedBodyText { get; init; }
```

Comment:
Please add an extra line here. Why are you not just call it body btw: And Why do you need it? The text gives you the opportunity to actually control the formatting. Is it not superior?

Reply:
Sure, thank you!
Text is like subject, body is like content.
The Discord Embed messages allow you to set the title and then the body. If user doesn't provide the body, only the title is set.